repo/packages/S/spark/12/marathon.json.mustache

```diff
@@ -43,7 +64,27 @@
             "image": "{{resource.assets.container.docker.spark_docker}}",
             "network": "HOST",
             "forcePullImage": true
-        }
+        },
+        "volumes": [
+            {
+                "hostPath": "/run/dcos/pki/",
+                "containerPath": "/run/dcos/pki/",
+                "mode": "RO"
```

Comment:
Technically, yes, but mustache templating blows and I can't get the comma situation right if I conditionally include all the volumes.  Including this volume won't cause any problems though.